lib/screens/authenticate/register.dart

```diff
@@ -80,31 +208,36 @@ class _RegisterState extends State<Register> {
                           if (_formKey.currentState.validate()) {
                             setState(() => loading = true);
                             try {
-                              await _auth.registerWithEmailAndPassword(
-                                  email, password);
+                              setState(() {
+                                email = email.trim();
+                              });
```

Comment:
Why using setState only to trip an email. Just trim it while sending a request. setState is expensive as it changes resources on screen

Reply:
fixed